README.md

```diff
@@ -888,6 +888,18 @@ unaware of any which supports _all_ of the features which this library supports.
 If you need help, please fork [this CodeSandbox][help-sandbox] and bring it up in
 [the chat][chat]
 
+## Swag
+
+Order a Glamorous T-Shirt! For every shirt sold, $1 will be donated to https://girlswhocode.com.
```

Comment:
Could we also update the wording to be more clear about things? For legal reasons, we want it to be clear that PayPal is not making any money here and is not administering the product. So wording that communicates that would be helpful. Maybe something like:

> A community member created this awesome Glamorous T-Shirt!

You could even use your wife's name and link to her portfolio or something if you like :)